subsys/net/l2/wifi/wifi_shell.c

```diff
@@ -717,7 +717,7 @@ static int cmd_wifi_twt_setup_quick(const struct shell *sh, size_t argc,
 	params.flow_id = 0;
 	params.setup.responder = 0;
 	params.setup.implicit = 1;
-	params.setup.trigger = 1;
+	params.setup.trigger = 0;
```

Comment:
Please change the commit title, `Seeing issues with trigger enabled TWT` is not really useful for understanding the change, just mention you change the default.